packages/core/src/test_resources/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
```

Comment:
I am surprised that this type hasn't already been declared and can't be reused. Could be mistaken but wanted to point it out thanks

Reply:
This type is intended to generically represent the different User types used across packages in the agent runtime based on whether we're using sqlite, supabase or sqljs. I couldn't find an abstract type that currently exists for this, but I'd be happy to replace it here if there is one. If there isn't one then I agree this would be helpful to have a real type that does this instead of this one sitting in a test file :)